cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -294,10 +294,7 @@ func KnownControllers() []string {
 	return ret.List()
 }
 
-var ControllersDisabledByDefault = sets.NewString(
-	"bootstrapsigner",
-	"tokencleaner",
-)
+var ControllersDisabledByDefault = sets.NewString()
```

Comment:
But we can't know that from the controller-manager side?
Also, using BTs for authn/authz is only half of the functionality.
BTs can be used for the verification part alone as well.

Do you have any counter-proposal? This was what has been discussed and approved in sig-cluster-lifecycle.